mysql-test/suite/galera/t/galera_alter_add_constraint.test

```diff
@@ -0,0 +1,23 @@
+# Test that contraint check is done on source and replicated node
+# and produces the same error, which leaves the cluster in consistent
+# state after consistency voting.
+
+--source include/galera_cluster.inc
+
+--connection node_2
+CALL mtr.add_suppression("Replica SQL: Error .Check constraint .CHK_i. is violated");
+--connection node_1
+CALL mtr.add_suppression("Replica SQL: Error .Check constraint .CHK_i. is violated");
+
+
+CREATE TABLE t (id INT PRIMARY KEY, i INT NOT NULL);
+
+INSERT INTO t VALUES(1, 100);
+
+# This will replicate before local execution.
+# Then it will fail locally, as the constraint check fails.
+# It is expected that the replica will fail as well with the same error.
+--error ER_CHECK_CONSTRAINT_VIOLATED
+ALTER TABLE t ADD CONSTRAINT CHK_i CHECK (i <=75);
```

Comment:
Can you add the test scenario covered in https://jira.percona.com/browse/PXC-4337 and https://jira.percona.com/browse/PXC-4338 as well?

Reply:
done